src/conduit/ConduitClient.php

```diff
@@ -203,19 +203,45 @@ private function addWonkaClaimToCookies($cookies) {
   private function getWonkaClaim() {
     $claim = '';
     // if the file exists and is current, use it.
-    $claim_file = getenv('ARC_USSO_COOKIE');
-    if (file_exists($claim_file)) {
-      $stale = time() - filemtime($claim_file);
-      if ($stale < 60) {
+    // append the pid
+    $claim_file_head = getenv('ARC_USSO_COOKIE');
+    $claim_file = sprintf('%s_%d', $claim_file_head, getmypid());
+    // Happy path
+    $claim = $this->happyPath($claim_file);
+
+    $now = time();
+    $wonka_cli = getenv('ARC_WONKA_CLI');
```

Comment:
But If I put it in a config file, anything can be provided there as well. Specifying a binary doesn't really change anything either since I can put anything I want in the binary. 

It's only a security issue if there is way for someone other than the user to shoot themselves in the foot, there's no way to stop users from shooting themselves in the foot if they choose. 

I need to run an external command and put those results in a cache file; using ENV variables for this makes the configuration flexible. Additionally, it has actual security benefits over a config file as in general people are more aware of managing permissions correctly on shell startup files. 

Putting configuration data in ENV variables is standard practice ( See 12 Factor App). 
However, please suggest an alternative to getenv and appropriate config file.